src/plugin.json

```diff
@@ -17,18 +14,18 @@
       "large": "img/clock.svg"
     },
     "links": [
-      {"name": "Project site", "url": "https://github.com/grafana/clock-panel"},
```

Comment:
Please avoid formatting changes that are not relevant to the code change. In this case it would be everything besides `skipDataQuery`